static/styles/left_sidebar.css

```diff
@@ -508,6 +508,10 @@ li.expanded_private_message {
     margin-top: 3px;
 }
 
+.stream-list-filter {
+    width: 90%;
+}
```

Comment:
Can you compare this to the right sidebar?

Reply:
We use bootstrap class for right sidebar `user-list-filter` and it works perfectly fine there.
```
input,
textarea,
.uneditable-input {
  width: 206px;
}
```
But `width: 206px` is not suitable in left-sidebar `stream-list-filter`, so we need to define "width" property seperately for `stream-list-filter`.

Reply:
We can use either `width: 236px` or `width: 90%`. Do I need to change it to 236px?

Reply:
Done.